### PR TITLE
FE-524

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Some changes needed @ipanagiv , please also check these along with my comments below:
1. Add a contact support prompt in [that one](https://docs.weatherxm.com/wifi-m5-bundle/m5-troubleshooting#one-or-more-variables-are-extremely-high-low-or-constant-for-long-periods-of-time) to contact support through https://support.weatherxm.com if none of the above work.

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx [hunk 1]
```diff
@@ -174,6 +174,56 @@ d. Top cap in white is the rest of M5 Miner.
 </div>
 </details>
 
+
+## Other Issues
+
+### M5 not working in 5 GHz WiFi.
+M5 is designed to work **only** at the 2.4GHz frequency. Please make sure the 2.4Ghz band on your router is enabled. If you are still experiencing issues, please contact us through [support](https://support.weatherxm.com)
+
+<hr />
```
Comment (PavlosTze) [inline docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx:183]: This should be moved above under the WiFi Connectivity section. Also add a dot at the end please.

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx [hunk 2]
```diff
@@ -174,6 +174,56 @@ d. Top cap in white is the rest of M5 Miner.
...
+  
+Please do not restart it and contact us through [support](https://support.weatherxm.com) to debug it and find the reason of the issue
+
+<hr />
+
+### The GPS sign on my M5 is red.
+
+Please check that the plug is firmly placed, then move the M5 with the GPS antenna outside for some minutes and place it close to a window. 
+
+<hr />
```
Comment (PavlosTze) [inline docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx:195]: Duplicate, there is another answer above in the GPS Location section regarding it.

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx [hunk 3]
```diff
@@ -174,6 +174,56 @@ d. Top cap in white is the rest of M5 Miner.
...
+
+
+### Pressure on M5 appears to be very low relatively to other reference stations of national or private networks etc. 
+
+For now we present only the absolute pressure and not the Mean Sea Level Pressure (MSLP). That means that the displayed pressure are affected by the elevation of your area. We work to consider the elevation given by your GPS in order to show the MSLP, which is commonly used.
+
+<hr />
```
Comment (PavlosTze) [inline docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx:202]: Let's move it above under the "Weather Station" section as more weather-variables-related issues are there.

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx [hunk 4]
```diff
@@ -174,6 +174,56 @@ d. Top cap in white is the rest of M5 Miner.
...
+
+Please disconnect the M5 from the power source, let it turn off by itself and reconnect it after 15-20 minutes. You can also try using a different USB cable and another wall adapter. If none of the previous work, try removing the four screws of the M5 bundle, separate the stacks and tighten the screws again.
+
+<hr />
+
+### I don't see the Wi-Fi name "WXM MinerM5" for initial setup on the app.
+
+It might take some time and attempts for it to connect. Please make sure that your Wi-Fi supports 2.4 GHz signals. You may also need to try from a different phone. You can also watch our instructional [video](https://www.youtube.com/watch?v=sUJEwuFq1CE).
+
+<hr />
```
Comment (PavlosTze) [inline docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx:214]: This should be moved above under the WiFi Connectivity section.

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Except my comments below please also:
1. Add a prompt in [that one](https://docs.weatherxm.com/wifi-m5-bundle/m5-troubleshooting#one-or-more-variables-are-extremely-high-low-or-constant-for-long-periods-of-time) to contact support through https://support.weatherxm.com if none of the above work.
2. Add a paragraph [here](https://docs.weatherxm.com/wifi-m5-bundle/m5-troubleshooting#wifi--connectivity) regarding the WiFi Repeater, as mentioned in the previously deleted FAQ: "Usually using a WiFi repeater is not an issue, but in some cases it may affect your M5’s connection.".

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx [hunk 5]
```diff
@@ -174,6 +174,96 @@ d. Top cap in white is the rest of M5 Miner.
...
+
+Assuming that the M5 unit is powered on, this may be due to a bad internet connection or bad RF connection between the M5 and the outdoor station.  
+
+<hr />
+
+### How long does it take for my device to show up on the explorer?
+
+It takes up to 30 minutes for your device to show up on the explorer.
+
+
```
[the remark this answers is not in the document]
Comment (PavlosTze) [reply docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx:226]: @ipanagiv Also this hasn't been fixed yet, you need to remove it from here and add a new box after we inform the user that he claimed his device containing this information about the explorer, [here](https://docs.weatherxm.com/wxm-devices/wifi-m5-bundle/m5-connect-and-claim#confirm-station-installation-location--claim-your-device) and [here](https://docs.weatherxm.com/wxm-devices/helium-bundle/helium-connect-and-claim#set-weather-station-operation-frequency).

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx (hunk 4 above)
Comment (ipanagiv) [reply docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx:223]: done

File: docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx (hunk 2 above)
Comment (ipanagiv) [reply docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx]: removed

**Review by PavlosTze — APPROVED** [review]
Good job @ipanagiv , feel free to merge it! 🚀

**Pull request merged**